irma/cmd/server.go

```diff
@@ -116,6 +116,14 @@ func setFlags(cmd *cobra.Command, production bool) error {
 	flags.StringSlice("revoke-perms", nil, "list of credentials that all requestors may revoke")
 	flags.Bool("skip-private-keys-check", false, "whether or not to skip checking whether the private keys that requestors have permission for using are present in the configuration")
 	flags.String("static-sessions", "", "preconfigured static sessions (in JSON)")
+
+	flags.String("store-type", "memory", "specifies how session state will be saved on the server, defaults to memory")
```

Comment:
removed the default here, added a fallthrough case in the switch so empty strings will use the memory case, too.

Reply:
The fall through case is also nice, when using the IRMA server as a Go library.

The fall through case will not break when also using Viper's own feature for default values as well?
Like this:
```
flags.String("store-type", "memory", "specifies how session state will be saved on the server")flags.String("store-type", "memory", "specifies how session state will be saved on the server")
```
Will lead to:
```
--store-type string                specifies how session state will be saved on the server (default "memory")
```

Reply:
I considered this solution and had discussed it with Sietse in person. Eventually I chose not to do so as I think it's clearer for the reader. Assume you want to change the default. You find the default set in this `flags.String` method and adjust it. You think you solved the problem but the default is additionally set in the case and then there would be two different defaults depending on if you use the library or don't. This scenario I'd like to avoid. Therefore it's now only encoded in the switch statement.